From bd6a5782f91158559e5ac622cedc9375486bc0e1 Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Tue, 29 May 2018 21:46:27 +0100
Subject: [fix] hackerrank, 'strong password', c# solution

-dash position in special characters list, protect against range
interpretation
---
 .../strong.password.cs                                      | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/hackerrank/algorithms.-.strings.-.easy.-.strong.password/strong.password.cs b/hackerrank/algorithms.-.strings.-.easy.-.strong.password/strong.password.cs
index e81b8c0..a1217a6 100644
--- a/hackerrank/algorithms.-.strings.-.easy.-.strong.password/strong.password.cs
+++ b/hackerrank/algorithms.-.strings.-.easy.-.strong.password/strong.password.cs
@@ -17,11 +17,7 @@ class Solution {
     // Complete the minimumNumber function below.
     static int minimumNumber(int n, string password) {
         // Return the minimum number of characters to make the password strong
-        var special_characters = "!@#$%^&*()-+";
-
-        // min length 6 means..
-
-        // # as n >= 1
+        var special_characters = "!@#$%^&*()+-"; // position of dash matters!
 
         // initially
         var toAdd = 0;
-- 
2.23.0

